File: examples/csharp-game/components/BoidVel.cs
// Boid's velocity half — the second name in scripts/Swarm.cs's batch (see
// components/Boid.cs; vx < vy keeps declared order == sorted == stream
// order).
[LabelleComponent]
record BoidVel
{
    public double vx = 0.0;
    public double vy = 0.0;
}

File: examples/rust-game/components/boid_vel.rs
// Boid's velocity half — the second name in scripts/swarm.rs's batch
// (see components/boid.rs; vx < vy keeps declared order == sorted ==
// stream order).
labelle::component! {
    BoidVel {
        vx: f32 = 0.0,
        vy: f32 = 0.0,
    }
}
